spec/support/forms/rendered_inline_form.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+RSpec.shared_context "with rendered inline form" do
+  include ViewComponent::TestHelpers
+
+  let(:form_arguments) { { url: "/foo", model: } }
+
+  def render_inline_form(&)
```

Comment:
As far as I can see, this example is not used at all. I admit that it is rather annpying, that the IDE identifies this method as the definition of `render_inline_form` (and not the Primer one) and thus always shows an error. So maybe it is better to remove this file.

<img width="555" height="183" alt="Image" src="https://github.com/user-attachments/assets/f21b6fe0-9322-4155-84d1-0da4f2f72888" />